src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -194,6 +194,16 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _autoScrollTimer.Tick({ this, &TermControl::_UpdateAutoScroll });
 
         _ApplyUISettings(_settings);
+
+        // Set the foreground and background here in the constructor
+        // (they will also be called later in UpdateAppearance, but we want to have it here so
+        //  that the terminal is not gray on startup)
+        const auto bg = _settings.DefaultBackground();
+        _BackgroundColorChanged(bg);

```

Comment:
I'm actually.. afraid of this. BackgroundColorChanged _is also a fire_and_forget_ which we should not use from the constructor (!)

Reply:
Ah, thanks for pointing that out. Fixed it here and in `_UpdateAppearanceFromUIThreadUnderLock`